### PR TITLE
Seedy-Threepio.Theseus2000 version 3.6C

### PR DESCRIPTION
**Pull request by seedy60**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/108193)

### CODE REVIEW

File: manifests/s/Seedy-Threepio/Theseus2000/3.6C/Seedy-Threepio.Theseus2000.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Seedy-Threepio.Theseus2000
+PackageVersion: 3.6C
+PackageLocale: en-US
+Publisher: Seedy-Threepio
+PackageName: Theseus2000
```
Comment (stephengillie) [inline manifests/s/Seedy-Threepio/Theseus2000/3.6C/Seedy-Threepio.Theseus2000.locale.en-US.yaml:8]:
```suggestion
PackageName: Theseus2000
PackageUrl: https://t2000.seedy.gq/
```

**Comment by stephengillie** [comment]
This package's main application is an interactive console: 
image

File: manifests/s/Seedy-Threepio/Theseus2000/3.6C/Seedy-Threepio.Theseus2000.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Seedy-Threepio.Theseus2000
+PackageVersion: 3.6C
+PackageLocale: en-US
+Publisher: Seedy-Threepio
```
Comment (Trenly) [inline manifests/s/Seedy-Threepio/Theseus2000/3.6C/Seedy-Threepio.Theseus2000.locale.en-US.yaml:7]:
```suggestion
Publisher: SeedyThreeSixty
```
To match control panel

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by ItzLevvie** [comment]
Close with reason: Stale pull request;

**Pull request closed without merging**